website/client/src/components/achievements/levelUp.vue

```diff
@@ -1,109 +1,197 @@
-        @click="close()"
-      >
-        {{ $t('onwards') }}
-      </button>
-      <br>
-      <!-- @TODO: Keep this? .checkboxinput(type='checkbox', v-model=
+      <span
+        class="star-group"
+        v-html="starGroup"
+      ></span>
+    </section>
+
+    <hr>
+    <p class="text">
+      {{ $t('levelup') }}
+    </p>
+
+    <section
+      v-if="displayRewardQuest"
+      class="greyed"
+    >
+      <div class="your-rewards flex">
+        <span
+          class="sparkles"
+          v-html="sparkles"
+        ></span>
+        <span class="text">{{ $t('yourRewards') }}</span>
+        <span
+          class="sparkles mirror"
+          v-html="sparkles"
+        ></span>
+      </div>
+
+      <div :class="questClass"></div>
+      <hr>
+    </section>
+    <!-- @TODO: Keep this? .checkboxinput(type='checkbox', v-model=
 'user.preferences.suppressModals.levelUp', @change='changeLevelupSuppress()')
 label(style='display:inline-block') {{ $t('dontShowAgain') }}
       -->
-    </div>
   </b-modal>
 </template>
 
 <style lang="scss">
   #level-up {
-    h2 {
-      color: #4f2a93;
+    @media (min-width: 576px) {
+      .modal-sm {
+        max-width: 330px;
+      }
+    }
+
+    header {
+      padding: 0;
+      border: none;
+
+      h5 {
+        margin: 31px auto 16px auto;
+        color: #4f2a93;
+      }
+
+      button {
+        position: absolute;
+        right: 18px;
+        top: 12px;
+      }
+    }
+
+    section {
+      width: 100%;
+      margin: 0;
+    }
+
+    hr {
+      margin: 0;
+      border-top: 25px solid rgb(255 63 249 / .2);
+    }
+
+    footer {
+      padding: 0;
+      border: none;
+
+      button {
+        margin: 0 auto 32px auto;
+      }
+    }
+
+    .flex {
+      display: flex;
     }
 
-    .modal-content {
-      min-width: 28em;
+    .greyed {
+      background-color: #f9f9f9;
     }
 
     .modal-body {
-      padding-top: 1em;
+      padding: 0;
     }
 
-    .modal-footer {
-      margin-top: 0;
+    .mirror {
+      transform: scaleX(-1);
     }
 
-    .herobox {
-      margin: auto 8.3em;
-      width: 6em;
-      height: 9em;
-      padding-top: 0;
-      cursor: default;
+    .star-group {
+      margin: auto;
+
+      svg {
+        height: 64px;
+        width: 40px;
+      }
     }
 
-    .character-sprites {
-      margin: 0;
-      width: 0;
+    .avatar {
+      cursor: auto;
+    }
+
+    .class-badge {
+      display: none !important;
     }
 
     .text {
-      font-size: 14px;
-      text-align: center;
-      color: #686274;
-      margin-top: 1em;
-      min-height: 0px;
+      margin: 0 24px 24px 24px;
+      min-height: auto;
     }
-  }
-</style>
 
-<style scoped>
-  .avatar {
-    margin-left: 6.8em;
+    .your-rewards {
+      margin: 0 auto;
+      width: fit-content;
+
+      .sparkles {
+        width: 32px;
+        margin-top: 12px;
+      }
+
+      .text {
+        font-weight: bold;
+        margin: 16px;
+        color: #4e4a57;
```

Comment:
for this and the other colors, can we use the variables (when they already exist) defined in colors.scss like here https://github.com/HabitRPG/habitica/blob/develop/website/client/src/components/chat/chatMessages.vue#L98 for example

Reply:
Awesome, will check it out. Thanks.

Reply:
Done.